### PR TITLE
Add continuous deployment

### LINKED ISSUE #11: Why can't you package this up and publish on Melpa?
I know you've said in the README the nature of this package makes it difficult, but there's no explanation. There are lots of packages on Melpa that have multiple files, and the process for creating them is clearly documented in the Elisp manual and Melpa. Can you explain what difficulty you have?

### PR DESCRIPTION
**Pull request by jscheid**
Also, bump version to 0.9.0.

Closes #11

### CODE REVIEW

**Comment by wyuenho** [comment]
Why are you still building tar balls, instructing users to install via Quelpa instead of Melpa, and deleted the lock file?

**Comment by jscheid** [comment]
1. Because they let myself and others test PRs easily, as described in `RELEASING.md`. If you mean the release tarball, it can't hurt to have a secondary source for installation/reference, uploading is 10 LOC.
2. Because it's not on Melpa yet, getting it on there is a separate step after which the instructions can be removed obviously.
3. I don't want to have to worry about keeping two lock files in sync, one is enough. I prefer yarn personally but that would be an extra dependency to install in CI, npm will do just fine.

**Comment by wyuenho** [comment]
Sounds reasonable

**Comment by jscheid** [comment]
Thanks for the review @wyuenho.

@asbish @aspiers any concerns or suggestions from your side? If there aren't any, I'll merge and then, in the not too distant future, submit a PR against MELPA - next weekend perhaps.

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        run: |
+          set -eou pipefail
+          echo ::set-env name=commit_message::$(
+            curl \
+              --silent \
+              --show-error \
+              --header "Accept: application/vnd.github.v3+json" \
+              --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+              "https://api.github.com/repos/${{ github.repository }}/commits/${{ github.sha }}" \
+            | jq --raw-output .commit.message
```
Comment (asbish) [inline .github/workflows/cicd.yml:123]: Although I'm not familiar with Github Actions or don't run the deploy action, I think `sudo apt-get install jq` might be needed before using `jq`.

Comment (jscheid) [reply .github/workflows/cicd.yml:123]: Thanks for taking a close look. I've tested this in a private repository and it was fine, I assume the ubuntu base image already contains it. Going to merge this now, thanks again.

**Comment by aspiers** [comment]
Sorry I didn't have time to review, but very happy you are making progress with this! :+1:

**Pull request merged**